patsylearn/patsy_adaptor.py

```diff
@@ -295,7 +336,4 @@ def transform(self, data):
         data : dict-like (pandas dataframe)
             Input data. Column names need to match variables in formula.
         """
-        if self.return_type == 'dataframe':
-            return dmatrix(self.design_, data, return_type='dataframe')
-        else:
-            return np.array(dmatrix(self.design_, data))
+        return dmatrix(self.design_, data, return_type=self.return_type)
```

Comment:
Will this work with pipelines? The idea was to return a numpy array as that is the default language of scikit-learn.